src/widgets/events/ui/events.tsx

```diff
@@ -34,7 +34,7 @@ export const Events = () => {
         </section>
 
         <section className={cx('cards')}>
-          {!actualEvents.length && (<Image src={photo3} alt="Community event" className={cx('event-img')} />)}
+          {!actualEvents.length && (<Image src={photo3} alt="Speaker presenting at a event" className={cx('event-img')} />)}
```

Comment:
Please, place it on 2 lines (2nd line begins from`&&`)

Reply:
Done